package.json

```diff
@@ -17,13 +17,15 @@
   "contributors": [],
   "license": "OGL-UK-3.0",
   "dependencies": {
-    "@hapi/hapi": "20.2.2",
-    "applicationinsights": "2.3.3",
     "@azure/identity": "2.0.5",
     "@azure/storage-blob": "12.10.0",
+    "@hapi/hapi": "20.2.2",
+    "applicationinsights": "2.3.3",
     "ffc-messaging": "2.5.1",
     "joi": "17.6.0",
     "log-timestamp": "0.3.0",
+    "moment": "^2.29.4",
```

Comment:
remove caret

Reply:
A quick google tells me I can run this and stop me having to check this every time!

`npm config set save-exact true`